app/controllers/spree/paybright_controller.rb

```diff
@@ -52,8 +56,17 @@ def handle_callback_params!
         response_code: paybright_params[:x_gateway_reference],
         amount: paybright_params[:x_amount]
       )
-      @payment.complete!
-      advance_and_complete(@payment.order)
+
+      begin
+        @payment.complete!
+        advance_and_complete(@payment.order)
+      rescue StandardError
+        if @payment.response_code.present?
+          @payment.void
+        end
+
+        return [false, 'Something went wrong with the order. Your Paybright application has been voided. Try to order again.']
```

Comment:
Since this looks like a user-facing message, can you make this an I18n key (with the above string as the default value)?

Reply:
sure thing